### DISCUSSION TITLE
JetBrains RubyMine not cycling through inline suggestions

### DISCUSSION THREAD

**Discussion by jsrn** [community/community#7006] [Copilot Conversations] [answered]
I'm testing the RubyMine integration and am unable to cycle through inline suggestions.

Keybindings (changed from default due to conflict with system keybindings):

<img width="633" alt="image" src="https://user-images.githubusercontent.com/1489178/139447733-193b1eb3-4227-4633-9eab-2abf18ceb57b.png">

Inline suggestion with side panel open to demonstrate that there are multiple alternatives:

<img width="1196" alt="image" src="https://user-images.githubusercontent.com/1489178/139448049-a7588cfe-558c-497c-8ad3-d09d83813283.png">

I would expect `ctrl` + `[`/`]` to cycle through suggestions in-line at this point, but it doesn't. Am I doing something wrong?

Thanks!

**Comment by jansorg** [comment, marked as answer]
Cycling through in-line suggestions is currently only available for Python and Java. One of the next updates will have this for other files, too.
